README.md

```diff
@@ -2139,7 +2139,7 @@ $deals->listProductsAttachedToADeal($collect);
 ```
 
 
-### <a name="add_a_product_to_the_deal_eventually_creating_a_new_item_called_a_deal_product"></a>![Method: ](https://apidocs.io/img/method.png ".DealsController.addAProductToTheDealEventuallyCreatingANewItemCalledADealProduct") addAProductToTheDealEventuallyCreatingANewItemCalledADealProduct
+### <a name="add_a_product_to_the_deal_eventually_creating_a_new_item_called_a_deal_product"></a>![Method: ](https://apidocs.io/img/method.png ".DealsController.addAProductToTheDealEventuallyCreatingANewItemCalledADealProduct") addAProductToTheDealEventuallyCreatingANewItemCalledADealProduct <br>_alias_ `addAProductToADeal`

```

Comment:
I cannot add this comment to the correct place 😅, but I was thinking that maybe it would be user-friendly to update the Example usage section as well: `$deals->addAProductToADeal($collect);`. Maybe for "backward compatibility" it is needed to keep the current longer version as well (i.e as a comment on the other line).